lib/client.c

```diff
@@ -1859,6 +1869,7 @@ SF_STATUS STDCALL _snowflake_execute_ex(SF_STMT *sfstmt,
     if (request(sfstmt->connection, &resp, queryURL, url_params,
                 url_paramSize , s_body, NULL,
                 POST_REQUEST_TYPE, &sfstmt->error, is_put_get_command)) {
+        log_info("Query execution receives response");
```

Comment:
I'm trying to be super safe and have all bases covered, with that in mind is it possible that the execution may get stuck anywhere between this point and `log_info("Query execution done");`?

Reply:
The only space I could image is that I point out in previous comment for the dead lock issue. But I could not image the case, but we still could add logs as check-points. Please check my first comments.